dev/io.openliberty.restfulWS.3.1_fat/bnd.bnd

```diff
@@ -0,0 +1,44 @@
+#*******************************************************************************
+# Copyright (c) 2020, 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src
+
+javac.source: 11
+javac.target: 11
+
+	
+fat.project: true
+
+# These features get added programmatically
+tested.features: \
+  restfulwsclient-3.1,\
+  restfulws-3.1,\
+  jsonp-2.1,\
+  cdi-4.0,\
+  connectors-2.1,\
+  jsonb-3.0,\
+  expressionlanguage-5.0,\
+  xmlbinding-4.0
```

Comment:
```suggestion
  jsonb-3.0
```
I don't believe these others are touched.  In reality tested.features can be removed completely since you don't do repeats.  Nothing is added programmatically.

Reply:
Done.